GlobalSync-frontend/README.md

```diff
@@ -0,0 +1,94 @@
+# GlobalSync <Nextjs/Typescript>
+
+**GlobalSync** is an innovative platform designed to empower remote teams to work seamlessly across different time zones. By providing intuitive tools for time zone coordination, collaboration, and scheduling, GlobalSync ensures productivity and convenience for global teams.
+
+# 🚀 Getting Started  
+
+### Open Using Daytona  
+
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
+2. **Create the Workspace**:  
+   ```bash  
+   daytona create https://github.com/AkashJana18/sample-nextjs-aceternity.git
```

Comment:
The URL is wrong here. It needs to be of this repo (https://github.com/daytonaio/sample-nextjs-aceternity)